src/test/groovy/org/prebid/server/functional/tests/privacy/GppAmpSpec.groovy

```diff
@@ -74,12 +75,16 @@ class GppAmpSpec extends PrivacyBaseSpec {
         storedRequestDao.save(storedRequest)
 
         when: "PBS processes amp request"
-        defaultPbsService.sendAmpRequest(ampRequest)
+        def ampResponse = defaultPbsService.sendAmpRequest(ampRequest)
 
         then: "Bidder request shouldn't contain regs.gpp"
         def bidderRequests = bidder.getBidderRequest(ampStoredRequest.id)
         assert !bidderRequests.regs.gpp
         assert !bidderRequests.regs.gppSid
+
+        and: "Repose should contain warning"
+        assert ampResponse.ext?.warnings[PREBID]*.code == [999]
+        assert ampResponse.ext?.warnings[PREBID]*.message[0].startsWith("Failed to parse gppSid: \'${gppSids}\'")
```

Comment:
nitpick: pls add a check for warnings in the success cases so we can ensure no warnings were accidentally included